docker/package/model.py

```diff
@@ -644,13 +646,22 @@ def gen_makefile(self, out):
 
 tezos-baking:
 
+wizard-structure:
+	mv $(CURDIR)/wizard_structure.py $(CURDIR)/wizard_structure.py
```

Comment:
:monkahmm:

Reply:
:laughing: Def. What would you suggest?

Reply:
Is `wizard-structure` target really needed?
I thnk you can safely omit it

Reply:
Moreover, seems like it actually causes `tezos-baking` binary package build to fail:
```
mv: '/tezos-packaging/docker/tezos-baking-12.0-rc2/wizard_structure.py' and '/tezos-packaging/docker/tezos-baking-12.0-rc2/wizard_structure.py' are the same file
make[1]: *** [Makefile:9: wizard-structure] Error 1
make[1]: *** Waiting for unfinished jobs....
```